app/src/shared/components/MobileNav/index.jsx

```diff
@@ -39,6 +39,24 @@ const MobileNav = compose(
         setOpen((current) => !current)
     }, [])
 
+    const siteSection = useMemo(() => {
+        if (redirect) {
+            if (redirect === '/' || redirect.startsWith('/marketplace')) {
+                return 'marketplace'
+            }
+
+            if (redirect.startsWith('/core') || redirect.startsWith('/canvas') || redirect.startsWith('/dashboard')) {
```

Comment:
Should there be an `editor` return for Canvases?

Reply:
Hmm, yes there should 👍 will fix